functions.php

```diff
@@ -935,9 +935,18 @@ function addToMailQueue($mysqli, $data) {
         $recipient = strval($email['recipient']);
         $recipient_name = strval($email['recipient_name']);
         $subject = strval($email['subject']);
-        $body = strval($email['body']);
-
-        mysqli_query($mysqli, "INSERT INTO email_queue SET email_recipient = '$recipient', email_recipient_name = '$recipient_name', email_from = '$from', email_from_name = '$from_name', email_subject = '$subject', email_content = '$body'");
+        $body = nullable_htmlentities($email['body']);
```

Comment:
@o-psi leave this as strval we SQL escape it before it goes to the function. Adding the htmlentities can break formatting while escaping SQL. HTML Entities is mainly reserved for output to prevent XSS. Email Clients such as outlook already do this.

Reply:
Gotcha, that makes sense now that you say that.